roles/provision-xamarin-app/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+# Creates a xamarin app representation using the mobile cli
+- name: "Create xamarin app represenation"
+  shell: mobile create client {{ appName }} xamarin {{ appIdentifier }} --namespace={{ namespace }}
```

Comment:
has this been tested? I did not think the mobile CLI would be available inside APBs.

Reply:
I did not notice that this was copied in, in the Dockerfile